runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -81,6 +81,11 @@
         /** True, if optimistic undo logging is disabled. */
         @Default boolean optimisticUndoDisabled = false;
 
+        /**
+         * Max size for a write request.
+         */
+        @Default int maxWriteSize = 0;
```

Comment:
Should we be checking if maxWriteSize is smaller than netty's max ByteBuf size? then we could perhaps print a warning indicating that maxWriteSize will be limited to netty's max ByteBuf Size...

Reply:
"is smaller than", typo ?

If the maxWriteSize is greater than netty's max size then the it will fail when the buffer is constructed.

Reply:
exactly sorry meant to say 'greater than'. So guess it makes no sense to print a warning and just let it fail.